app/experimenter/static/js/tests/AddonForm.test.js

```diff
@@ -21,6 +20,12 @@ import {
 } from "experimenter/tests/DataFactory";
 
 describe("The `DesignForm` component for Addon", () => {
+  beforeAll(() => {
+    Object.defineProperty(window, "location", {
+      writable: true,
+      value: { replace: jest.fn() },
+    });
```

Comment:
Looks like we have to do this pattern a bunch, can you pull it out into a util function somewhere?  Otherwise everything looks great!

Reply:
I moved it to `setup.js` not sure if this is the right way to go about it...